docroot/themes/humsci/humsci_basic/patterns/card/card.html.twig

```diff
@@ -0,0 +1,150 @@
+{#
```

Comment:
We have a module that integrates with the Decanter twig files and provides some Drupal glue code that could save you some time if you're interested. Eg: https://github.com/SU-SWS/jumpstart_ui/blob/8.x-1.x/templates/components/card/card.html.twig

https://github.com/SU-SWS/jumpstart_ui/

We plan on maintaining this module with the latest from Decanter.

Reply:
Thank you @sherakama. We actually pulled from that jumpstart_ui markup originally, but then realized we need to stick with the existing horizontal card, vertical_card etc from other sites for better consistency.